src/cgroups/v2/manager.rs

```diff
@@ -123,6 +123,10 @@ impl CgroupManager for Manager {
                 ControllerType::Io => Io::apply(linux_resources, &&full_cgroup_path)?,
                 ControllerType::Memory => Memory::apply(linux_resources, &full_cgroup_path)?,
                 ControllerType::Pids => Pids::apply(linux_resources, &&full_cgroup_path)?,
+                ControllerType::Unknown(tpe) => {
+                    log::warn!("Controller {} is not yet implemented.", tpe);
+                    continue;
```

Comment:
The 'continue' is not necessary

Reply:
Actually this never gets called, because CONTROLLER_TYPE is a constant list of values.

Reply:
You're right! I changed it to a simpler way of implementation.